base/compiler/ssair/driver.jl

```diff
@@ -34,15 +34,33 @@ function normalize(@nospecialize(stmt), meta::Vector{Any})
     return stmt
 end
 
-function convert_to_ircode(ci::CodeInfo, code::Vector{Any}, coverage::Bool, nargs::Int, sv::OptimizationState)
+function add_opaque_closure_argtypes!(argtypes, t)
```

Comment:
Seems like a few methods still need type annotations in the Core.Compiler module. I don't want to mark each case.

Aside: also seems like this particular one might be unused? but github might just be hiding the right file diff